pytx/settings/development.py

```diff
@@ -19,3 +19,7 @@
 MEDIA_URL = STATIC_URL + 'uploads/'
 
 STATICFILES_DIRS.append(MEDIA_ROOT)
+
+SECRET_KEY = "a insecure development secret"
```

Comment:
remove this and make a `.env` instead. .env will be loaded automatically

should look like:

```
SECRET_KEY=narf
```